### ISSUE TITLE
Skin Packs don't get added to resource_packs

### ISSUE THREAD

**Issue by ndtreviv** [cubeworx/mcbe-server#14]
We created a skin pack and put it in the `addons` directory. From the logs I can see that it is unzipping the zip file, but then it doesn't get copied into resource_packs or symlinked etc. 

There aren't any errors... 🤔 Do custom skins not work on this server?
I've attached the zip file for you to see.

[Admin_Skins.zip](https://github.com/cubeworx/mcbe-server/files/7797743/Admin_Skins.zip)

**Comment by claflico** [comment]
When I wrote those functions only resource packs and behavior packs were supported. Looks like skin packs are supported now so as long as they are packaged in a similar fashion it should be really easy for me to add them to the entry script. I'll use the one you attached to help test too.

https://docs.microsoft.com/en-us/minecraft/creator/documents/gettingstarted

**Comment by ndtreviv** [comment]
I read the docs and corrected the skin pack. This is the corrected version:
[Admin_Skins.zip](https://github.com/cubeworx/mcbe-server/files/7878034/Admin_Skins.zip)

I also forked this repo and modified the addons script to support skin packs. I can see from the script logs that they get symlinked in, but unfortunately I can't find them anywhere when I join the world on the Switch - I expected to see them under Global Resources, but they weren't there. :(

**Comment by claflico** [comment]
I experimented with this a couple of nights this week. I updated the scripts to support skin packs but could never get them to show up.

I searched quite a bit online and couldn't find a single tutorial for installing skin packs on a server but found multiple tutorials for installing them onto the local client by double clicking on the .mcaddon file.

Specifically for the Nintendo switch, the tutorials i found all said that they had to be installed via the marketplace. 

Given my testing and research I'm thinking that skin packs can't be installed via the server. If you figure out otherwise then I'll keep experimenting with it though.
